r/scripts/install_dependencies.R

```diff
@@ -82,7 +82,7 @@ install_arrow_version <- function(version_to_install) {
 }
 
 dependencies <- c("testthat", "bookdown", "knitr", "purrr", "remotes", 
-                  "dplyr", "stringr", "reticulate")
+                  "dplyr", "stringr", "reticulate", "duckdb")
```

Comment:
```suggestion
                  "dplyr", "stringr", "reticulate", "duckdb", "dbplyr")
```

I think we need this, due to failing test